src/download_and_filter.py

```diff
@@ -435,3 +436,4 @@ def extract_uids_and_filter_ids(group):
             savedir=args.savedir,
             debug=args.debug
         )
+        data_bibtex.generate_bibtex(collection_key, save_to_file=True, output_dir=args.savedir)
```

Comment:
Could we actually run this for all selected datasets together, up by where it generates the provenance card?
```
data_provenance_card.generate_datacard
```

I think that would be better as some users may not want to generate the data, but just the bibtex. Otherwise looks great!

Reply:
@shayne-longpre 
Thanks for the suggestion. I changed the code to generate BibTeX for all selected datasets together. Now creation of BibTeX is separate from data generation